test/helpers/TestHelper.sol

```diff
@@ -0,0 +1,282 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity 0.8.10;
+
+import "forge-std/Test.sol";
+
+import "src/LBFactory.sol";
+import "src/LBPair.sol";
+import "src/LBRouter.sol";
+import "src/LBQuoter.sol";
+import "src/LBErrors.sol";
+import "src/interfaces/ILBRouter.sol";
+import "src/interfaces/IJoeRouter02.sol";
+import "src/LBToken.sol";
+import "src/libraries/Math512Bits.sol";
+import "src/libraries/Constants.sol";
+
+import "test/mocks/WAVAX.sol";
+import "test/mocks/ERC20.sol";
+import "test/mocks/FlashloanBorrower.sol";
+import "test/mocks/ERC20TransferTax.sol";
+
+abstract contract TestHelper is Test, IERC165 {
+    using Math512Bits for uint256;
+
+    uint24 internal constant ID_ONE = 2 ** 23;
+    uint256 internal constant BASIS_POINT_MAX = 10_000;
+
+    // Avalanche market config for 10bps
+    uint16 internal constant DEFAULT_BIN_STEP = 10;
+    uint16 internal constant DEFAULT_BASE_FACTOR = 1000;
+    uint16 internal constant DEFAULT_FILTER_PERIOD = 30;
+    uint16 internal constant DEFAULT_DECAY_PERIOD = 600;
+    uint16 internal constant DEFAULT_REDUCTION_FACTOR = 500;
+    uint24 internal constant DEFAULT_VARIABLE_FEE_CONTROL = 4000;
+    uint16 internal constant DEFAULT_PROTOCOL_SHARE = 100;
+    uint24 internal constant DEFAULT_MAX_VOLATILITY_ACCUMULATED = 350_000;
+    uint16 internal constant DEFAULT_SAMPLE_LIFETIME = 120;
+
+    address payable internal immutable DEV = payable(address(this));
+    address payable internal constant ALICE = payable(0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266);
+    address payable internal constant BOB = payable(0x70997970C51812dc3A010C7d01b50e0d17dc79C8);
```

Comment:
I like to use `address(bytes20(keccak256("ALICE")))` instead of hardcodding some addresses

Reply:
Actually there is even better in Foundry now: `makeAddr("alice")`